.github/workflows/integration-tests-kubernetes-core.yaml

```diff
@@ -0,0 +1,143 @@
+---
+name: Integration tests
+on:
+  pull_request:
+    branches:
+      - main
+      - stable-*
+
+jobs:
+  splitter:
+    env:
+      source_dir: "./source"
+    runs-on: ubuntu-latest
+    outputs:
+      test_targets: ${{ steps.splitter.outputs.test_targets }}
+      test_targets_json: ${{ steps.splitter.outputs.test_targets_json }}
+      test_jobs: ${{ steps.splitter.outputs.test_jobs }}
+    steps:
+      - name: Checkout the collection repository
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-collections/kubernetes.core
+          ref: main
+          path: ${{ env.source_dir }}
+          fetch-depth: "0"
+
+      - name: Split all Kubernetes targets into Job
+        id: splitter
+        uses: ansible-network/github_actions/.github/actions/ansible_test_splitter@main
+        with:
+          collections_to_test: ${{ env.source_dir }}
+          total_jobs: 8
+        env:
+          ANSIBLE_TEST_ALL_THE_TARGETS: "true"
```

Comment:
Does this mean that it's always going to test all the targets?

Reply:
Yes, this is the intent. This is a cross-collection testing, the `cloud.common` collection does not define integration tests, the goal of the integration tests here is to validate the full `kubernets.core` test suite is not impacted by changes on this collection

Reply:
Ah, got it. Thanks!